.github/workflows/build-test-deploy.yml

```diff
@@ -823,6 +823,26 @@ jobs:
           command: nix -- build .#acala-node
           token: "${{ secrets.CACHIX_AUTH_TOKEN }}"
 
+  package-dali-subxt-client:
+    name: "Package Dali subxt client"
+    runs-on:
+      - self-hosted
+      - linux
+      - x64
+      - sre
+    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
```

Comment:
Shouldn't this be removed? Else we can merge code that breaks this flow?

Reply:
The task requires this.
But, you are right.
I removed this.